webapp/apps/taxbrain/tests/test_views.py

```diff
@@ -57,6 +57,20 @@ def test_taxbrain_post(self, data_source):
         check_posted_params(result['tb_dropq_compute'], truth_mods,
                             str(START_YEAR), data_source=data_source)
 
+    @pytest.mark.xfail
```

Comment:
@lucassz This shouldn't be xfail-ed. That means that `py.test` will ignore it if the test fails.

Reply:
@hdoupe Thank you. I think I had understood the purpose of `xfail`.